layer/filestore_test.go

```diff
@@ -150,3 +150,16 @@ func TestGetOrphan(t *testing.T) {
 		t.Fatalf("Expected to have one orphan layer")
 	}
 }
+
+func TestFileMetadataStore_StartTransaction(t *testing.T) {
+	fms, _, cleanup := newFileMetadataStore(t)
+	defer cleanup()
+
+	tx, err := fms.StartTransaction("test-start-transaction")
+	if err != nil {
+		t.Fatal(err)
+	}
+	defer tx.Cancel()
+
+	// TODO: Check that cache ID is persisted.
```

Comment:
nit: I'd either move the requirements to finish this test into this commit, or move the test into the second commit to avoid the need for an empty test. Alternatively, when this gets merged we can just squash the two commits into one.

Reply:
I added a test for `len(cacheID) == 0` branch here.